easybuild/easyconfigs/p/pyqstem/pyqstem-1.0.3-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = "PythonPackage"
+
+name = 'pyqstem'
+version = '1.0.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/jacobjma/PyQSTEM'
+description = """QSTEM is a program for quantitative image simulation in electron
+microscopy, including TEM, STEM and CBED image simulation.
+
+This project interfaces the QSTEM code with Python and the Atomic
+Simulation Environment (ASE) to provide a single environment for
+building models, simulating and analysing images."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('pyqstem-1.0.3-Remove-unused-scipy-import.patch', 1),
+]
+checksums = [
+    '612c4c95531ea9239c91e6f8b9e29a954ac873dd8584b6a57512dcfb6263ff0a',  # pyqstem-1.0.3.tar.gz
+    # pyqstem-1.0.3-Remove-unused-scipy-import.patch
+    'de1e6615057648c4d1f857f58d751699c1b292b4fe0c0f8b2f0b300e99f91d59',
+]
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('scikit-image', '0.14.1', versionsuffix),
+    ('ASE', '3.17.0', versionsuffix),
```

Comment:
@schiotz Using ASE 3.17.0 breaks the tests because other easyconfigs using a `2018b` toolchain use ASE 3.16.2 as dependency (and hence there's a risk of running into conflicts).

There are couple of options here:

* change the version here to 3.16.2
* add `-ASE-3.17.0` to the `versionsuffix`, similar to what we did for GPAW (cfr. #8751)
* bump this to a more recent toolchain where you could use the existing ASE 3.18.0 instead

Reply:
@boegel My fault!  I checked the ASE version in the wrong place, and thought we used 17.0 i 2018b.  The ASE version is completely uncritical for pyqstem, so I will just change the version and check in again.

I think ASE 3.19.0 is about to be released (next week, probably).  If a new GPAW also comes along not too long after, then I will make ASE and GPAW versions for the 2019b toolchains, but I hold it off a bit so we do not settle on an ASE version just before a new one is released.